sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/ShareAPITests.groovy

```diff
@@ -225,7 +223,7 @@ class ShareAPITests extends APISpec {
         when:
         def createSnapshotResponse = primaryShareClient.createSnapshotWithResponse(null, null, null)
         def shareSnapshotClient = new ShareClientBuilder().shareName(shareSnapshotName).connectionString(environment.primaryAccount.connectionString)
-            .snapshot(createSnapshotResponse.getValue().getSnapshot()).httpClient(new NettyAsyncHttpClientBuilder().build())
+            .snapshot(createSnapshotResponse.getValue().getSnapshot()).httpClient(getHttpClient())
```

Comment:
Something in the upgrade of either Jacoco or Surefire resulted in this line of code causing a deadlock